bus/src/main/java/org/hawkular/metrics/component/publish/InsertedDataSubscriber.java

```diff
@@ -64,8 +68,21 @@
 
     private Subscription subscription;
 
+    @Inject
+    private PublishCommandTable publish;
+
+    @Inject
+    @Configurable
+    @ConfigurationProperty(METRICS_PUBLISH_PERIOD)
+    private String publishPeriodProperty;
+    private int publishPeriod;
+
     public void onMetricsServiceReady(@Observes @ServiceReady ServiceReadyEvent event) {
-        Observable<List<Metric<?>>> events = event.getInsertedData().buffer(50, TimeUnit.MILLISECONDS, 100)
+        publishPeriod = getPublishPeriod();
+        Observable<List<Metric<?>>> events = event.getInsertedData()
+                .filter(m -> m.getType() != MetricType.STRING)
+                .filter(m -> publish.isPublished(m.getMetricId()))
```

Comment:
@lucasponce Is this comparing apples to apples?  is m.getMetricId() going to match the id placed in the cache by alerting?  It may be OK, just checking to make sure alerting is not using a type-prefixed value and this value perhaps not being type-prefixed.  Also asking whether tenant needs to be involved.